docs/content/release-notes/0.2.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "0.2.0"
+date: "2016-10-16"
+
+menu:
+  main:
+    parent: "release-notes"
+    identifier: 0.2.0
+    weight: 80
```

Comment:
probably don't need a weight here or in the other one. We're fine with sorting by reverse publish date.